views/mailinglistwidget.ejs

```diff
@@ -0,0 +1,44 @@
+<ul id="groups" class="group-list">
+	<li id="<%= groups[i].urlName %>">
+    <form action="/mailinglists/<%= groups[i].urlName %>" method="post">
+			<% if (emails.length == 1) { %>
```

Comment:
Update: removing this if branch makes the page work as expected for multiple email addresses